pages/wallet.vue

```diff
@@ -65,6 +69,17 @@ export default Vue.extend({
 
   beforeDestroy() {
     this.$accessor.wallet.resetPage()
+  },
+
+  methods: {
+    checkForReferralCode() {
+      const { $route } = this
+      const { code } = $route.query
+
+      if ($route.name === 'register' && code && code.toString().trim() !== '') {
```

Comment:
Why are we introducing `register` route here?

Reply:
@cmhchoi wanted the `register` route `/register?code=xxx` for referral links, so it looks nicer.

Reply:
Okay, thanks for clarifying. Can we introduce a flag similar to `META_TAGS_ENABLED` named `REFERRALS_ENABLED` and conditionally include this functionality so the relayers can decide whether to have it or not?